### PR TITLE
get backfill status api

### PR DESCRIPTION
**Pull request by hdou731**

### CODE REVIEW

File: client/src/main/proto/app/cash/backfila/service.proto [hunk 1]
```diff
@@ -75,3 +75,18 @@ message CreateAndStartBackfillRequest {
 message CreateAndStartBackfillResponse {
   optional uint64 backfill_run_id = 1;
 }
+
+// Checks on the status of a backfill.
+message CheckBackfillStatusRequest {
```
Comment (shellderp) [inline client/src/main/proto/app/cash/backfila/service.proto:80]: `Get` seems like our typical verb here instead of Check

Comment (hdou731) [reply client/src/main/proto/app/cash/backfila/service.proto:80]: I didn't want to have it confused with the dashboard action

Comment (shellderp) [reply client/src/main/proto/app/cash/backfila/service.proto:80]: oh I see

**Review by shellderp — APPROVED** [review]

**Comment by shellderp** [comment]
I'm curious what you'll use this for

File: client/src/main/proto/app/cash/backfila/service.proto (hunk 1 above)
Comment (hdou731) [reply client/src/main/proto/app/cash/backfila/service.proto:80]: > I'm curious what you'll use this for
Gonna use it to automate stock splits. Part of the procedure is to run a backfill in portfolios

File: client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/EmbeddedBackfila.kt [hunk 2]
```diff
@@ -66,6 +68,12 @@ internal class EmbeddedBackfila @Inject internal constructor(
     return Calls.response(CreateAndStartBackfillResponse(backfillId.toLong()))
   }
 
+  override fun checkBackfillStatus(request: CheckBackfillStatusRequest): Call<CheckBackfillStatusResponse> {
+    checkNotNull(request.backfill_run_id)
+    // TODO(hdou): actually check the status
```
Comment (mpawliszyn) [inline client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/EmbeddedBackfila.kt:73]: Are you working on a followup @hdou731 ?

**Pull request merged**